### PR TITLE
feat: add updater script content

### PR DESCRIPTION
**Pull request by davidzwa**
The script will uninstall FDMM, fetch the latest released version, update its packages, and install the service again.

### CODE REVIEW

File: src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/install-fdm-monster.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { Service } = require('node-linux');
 const { join } = require("path");
 
 // Create a new service object
-const rootPath = join("../fdm-monster/server/");
+const rootPath = "/home/pi/fdm-monster/server/";
```
Comment (mkevenaar) [inline src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/install-fdm-monster.js:5]: There is an option in CustomPiOS to change the username, AFAIK, therefore I used the join

File: src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/uninstall-fdm-monster.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const { Service } = require('node-linux');
 const { join } = require("path");
 
 // Create a new service object
-const rootPath = join(__dirname, "/pi/fdm-monster/server/");
+const rootPath = "/home/pi/fdm-monster/server/";
```
Comment (mkevenaar) [inline src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/uninstall-fdm-monster.js:5]: Same here

File: src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/update-fdm-monster.sh [hunk 3]
```diff
@@ -1,2 +1,61 @@
...
+if [ "$EUID" -ne 0 ]; then
+  echo "Please run as root, this script must reinstall a service and, therefore, requires elevated permissions (root)"
+  exit
+fi
+
+# Step 0b) Set the variables needed later
+ts=6 # total steps
+org=fdm-monster
+repo=fdm-monster
+server_path="/home/pi/fdm-monster/server/"
```
Comment (mkevenaar) [inline src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/update-fdm-monster.sh:28]: Use relative path here. 
```suggestion
server_path="~/fdm-monster/server/"
```

File: src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/update-fdm-monster.sh [hunk 4]
```diff
@@ -1,2 +1,61 @@
...
+daemon_path="/home/pi/fdm-monster-daemon/"
```
Comment (mkevenaar) [inline src/modules/monsterpi/filesystem/home/pi/fdm-monster-daemon/update-fdm-monster.sh:29]:
```suggestion
daemon_path="~/fdm-monster-daemon/"
```

File: src/modules/monsterpi/filesystem/home/pi/scripts/welcome [hunk 5]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+_NAME=$(hostname)
+_IP=$(hostname -I)
+_FDM_MONSTER_VERSION=$(jq -r .version /home/pi/fdm-monster/server/package.json || echo "unknown")
```
Comment (mkevenaar) [inline src/modules/monsterpi/filesystem/home/pi/scripts/welcome:5]:
```suggestion
_FDM_MONSTER_VERSION=$(jq -r .version ~/fdm-monster/server/package.json || echo "unknown")
```

**Comment by davidzwa** [comment]
Local build also completed in the green.

**Review by mkevenaar — CHANGES_REQUESTED** [review]

**Review by mkevenaar — APPROVED** [review]

**Pull request merged**